portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,67 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
     <script src="script.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+             <!--navbar-->
+<ul class="nav justify-content-center">
+  <li class="nav-item">
+    <a class="nav-link active" href=".center">HOME</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" href="#project">PROJECT</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" href="#Ab">ABOUT ME</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">CONTACT ME</a>
+  </li>
+</ul>
+<img class="center" src="/images/Clarke-Henry_Jewel-logo.png">
+    <h1> Hello! my name is Jewel!</h1>
+    <h2>I'm a Graphic Designer, Aspiring Web and Game developer.</h2>
+    <!-- projects -->
+
+    <h1 id="project" > My Projects...</h1>
+<div id="carouselExampleInterval" class="carousel slide" data-ride="carousel">
+  <div class="carousel-inner">
+    <div class="carousel-item active" data-interval="10000">
+      <img src="/images/TVXQ.png" class="d-block w-100" alt="...">
+    </div>
+    <div class="carousel-item" data-interval="2000">
+      <img src="/images/Zhoumi-.png" class="d-block w-100" alt="...">
+    </div>
+    <div class="carousel-item">
+      <img src="/images/color.png" class="d-block w-100" alt="Dummy pic">
+    </div>
+  </div>
+  <a class="carousel-control-prev" href="#carouselExampleInterval" role="button" data-slide="prev">
+    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+    <span class="sr-only">Previous</span>
+  </a>
+  <a class="carousel-control-next" href="#carouselExampleInterval" role="button" data-slide="next">
+    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+    <span class="sr-only">Next</span>
+  </a>
+</div>
+
+<h1> Other Projects</h1>
+    <h2>To be coutinue....</h2>
+
+<h1 id="Ab">About me</h1> 
+    <h2>To be coutinue...</h2> 
+
+
+
+    <!-- <div id="content">
```

Comment:
At least at Google, we prefer that commented out code be deleted instead -- you are keeping everything in Git so can always get it back from an older version.

Reply:
Thank you Mr. Kroeger.  I didn't realize I can refer back to previous version with git.

Reply:
Please call me "Rob". Mr Kroeger is my grandfather or my dad. 🙂

And regarding `git`: `git log` shows you all of the past commits and `git show <hash>` will show you any particular past commit.